pages/governance.md

```diff
@@ -0,0 +1,283 @@
+high-performance software stack for mathematical optimization.
+
+## Code of Conduct
+
+The JuMP community strongly values inclusivity and diversity. Everyone should
+treat others with the utmost respect. Everyone in the community must adhere to
+the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct) and the
+[Julia Community Standards](https://julialang.org/community/standards/), which
+reflect the values of our community. Both these links have information on how to
+confidentially report a violation.
+
+## Entities
+
+This section outlines the different entities that exist within the JuMP project,
+their basic role, and how membership of each entity is determined.
+
+### BDFL
+
+Due to his role in the creation of JuMP, [Miles Lubin](https://github.com/mlubin)
+holds the role of [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life).
+
+### Core Contributors
+
+Core contributors lead the technical development of the JuMP project, and they
+are the ultimate authority on the direction of the JuMP project.
+
+The current core contributors are:
+
+* [Miles Lubin (@mlubin)](https://github.com/mlubin)
+* [Benoît Legat (@blegat)](https://github.com/blegat)
+* [Joaquim Dias Garcia (@joaquimg)](https://github.com/joaquimg)
+* [Joey Huchette (@joehuchette)](https://github.com/joehuchette)
+* [Oscar Dowson (@odow)](https://github.com/odow)
+
+A new core contributor may be added by consensus of the current core
+contributors and notification to the [Steering Council].
+
+Before becoming a core contributor, it is expected that the community member is
+a [repository maintainer](#repository-maintainers) of several repositories.
+
+### Emeritus Core Contributors
+
+Emeritus core contributors are community members who were [core contributors],
+but have stepped back to a less active role.
+
+The emeritus core contributors are:
+
+* [Iain Dunning (@iainnz)](https://github.com/iainnz)
+
+A core contributor may choose to switch to emeritus status by informing the
+other [core contributors] and the [Steering Council].
+
+### Repository Maintainers
+
+Repository maintainers are trusted members of the community who help the [core
+contributors] by managing a limited number of repositories in the [JuMP-dev
+Github organization].
+
+Any core contributor has individual authority to appoint a community member as a
+repository maintainer of a repository with notification to the other [core
+contributors].
+
+Before becoming a repository maintainer, it is expected that the community
+member will have been an active participant in the development and maintenance
+of a repository for a sustained period of time. This includes triaging issues,
+proposing and reviewing pull requests, and updating any binary dependencies as
+needed.
+
+Community members may be maintainers of multiple repositories at the same time.
+
+Each repository should list the current maintainers in their corresponding
```

Comment:
This might create an artificial barrier. Some people might be good repository maintainers but not want to have their name on the README, which potentially invites attention and demands from users. We could use a separate and less user-facing page to list repository maintainers.

Reply:
Fair point. I've removed for now. We can revisit if/how to give maintainers recognition in future. Are you thinking a `MAINTAINERS.md` in each repo? Or a page on `jump.dev`?

Reply:
I was thinking a page on jump.dev.